docs/cmd/kn_service_update.md

```diff
@@ -71,6 +71,7 @@ kn service update NAME
       --requests-cpu string           DEPRECATED: please use --request instead. The requested CPU (e.g., 250m).
       --requests-memory string        DEPRECATED: please use --request instead. The requested memory (e.g., 64Mi).
       --revision-name string          The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
+      --scale int                     Minimal and Maximal number of replicas.
```

Comment:
Wonder if “Minimum” and “maximum” are better words here?

Reply:
:+1: (if I look at the result of Google hits for "minimum number" and "minimal number"). Also please use lower case, something like

"Minimum and maximum number of replicas"

Reply:
I had "Set min and max scale as the same value" but something recommended those changes when I submitted the first PR? I'll make this update.